src/services/services.ts

```diff
@@ -6118,6 +6138,90 @@ module ts {
                 }
             }
 
+            function classifyComment(token: Node, kind: SyntaxKind, start: number, width: number) {
+                if (kind === SyntaxKind.MultiLineCommentTrivia) {
+                    // See if this is a doc comment.  If so, we'll classify certain portions of it
+                    // specially.
+                    let jsDocComment = parseIsolatedJSDocComment(sourceFile.text, start, width);
+                    if (jsDocComment && jsDocComment.jsDocComment) {

```

Comment:
What is even going on here? What is `jsDocComment.jsDocComment`? When will it not be set?